src/WebJobs.Script/Host/ScriptHost.cs

```diff
@@ -267,9 +276,10 @@ public async Task InitializeAsync()
 
                 // Generate Functions
                 IEnumerable<FunctionMetadata> functions = GetFunctionsMetadata();
-                if (Utility.ShouldInitiliazeLanguageWorkers(functions, _currentRuntimelanguage))
+                _currentRuntimelanguage = _currentRuntimelanguage ?? Utility.GetCurrentLanguageRuntime(functions);
```

Comment:
These methods/variable names should really be about the "runtime stack" as opposed to language, as that's really what they represent.

Reply:
thanks! renaming where ever to workerRuntime